charts/ratify/templates/configmap.yaml

```diff
@@ -11,6 +11,9 @@ data:
       },
       "store": {
         "version": "1.0.0",
+        {{- if .Values.oras.useHttp }}
+        "useHttp": true,
```

Comment:
wonder if `"useHttp": {{ .Values.oras.useHttp }}` will work.

Reply:
good call! I will switch to this